services/venice-controller/src/main/java/com/linkedin/venice/controller/StoreBackupVersionCleanupService.java

```diff
@@ -39,8 +64,12 @@ public class StoreBackupVersionCleanupService extends AbstractVeniceService {
   private final Thread cleanupThread;
   private final long sleepInterval;
   private final long defaultBackupVersionRetentionMs;
+
+  private final Map<String, String> urlMap = new HashMap<>();
   private final AtomicBoolean stop = new AtomicBoolean(false);
 
+  private final CloseableHttpAsyncClient storeClient;
```

Comment:
`storeClient` name is confusing since it is not going to talk to any Venice store.
Maybe just call it `httpClient` instead?